appveyor.yml

```diff
@@ -1,31 +1,25 @@
-version: 1.0.{build}
 os: Visual Studio 2015
-before_build:
-- git submodule update --init
-- nuget restore Avalonia.sln
-
+platform:
+- Any CPU
+configuration:
+- Release
 environment:
-  myget_key:
+  NUGET_API_KEY:
+    secure: ADD_SECURE_NUGET_API_KEY_HERE
+  NUGET_API_URL: https://www.nuget.org/api/v2/package
+  MYGET_API_KEY:
     secure: XOgD5bJUKNOS2kDDgb+affS4pDcslxALh+xvvnr1Koy0PjXlhILsBdNhxRe0KcNm
```

Comment:
Oh! I didn't realise we had this in the repo. That's bad isn't it? Means someone could publish malicious packages to our MyGet feed.